test/tests/composer.js

```diff
@@ -1225,23 +1225,23 @@ module.exports = Nodal => {
             children__is_favorite: true,
             children__license: null,
             pets__name: 'Oliver',
-            pets__alive: true,
+            pets__is_alive: true,
             name: 'Zoolander',
             pets__animal__in: ['Cat']
           },
           {
             children__is_favorite: true,
             children__license__not_null: true,
             pets__name: 'Oliver',
-            pets__alive: true,
+            pets__is_alive: true,
```

Comment:
Add a test for comparators too (`pets__name__ilike` or `__lt`)

Reply:
Yeah I could add those if you like but this change was just to fix the bad column name used in the original test.